README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About Me
+
+Hello! My name is **Nikhil Mandalaparthy** (my first name is pronounced *nick-hill*). I'm currently a MPP student at the Harris School of Public Policy, and I just graduated in June from the College, with a BA in Public Policy Studies. This is my [personal website](https://nikhilmandala.com/).
+
+## What I Look Like
+
+![Me](Headshot - Cropped.jpg){ width=50% }
```

Comment:
The image did not render properly on GitHub. I suspect this is because you used a local file path. Be sure to preview submissions on GitHub before submitting them. Manipulating the width was a more sophisticated operation, which wasn't included in the provided Markdown syntax guide, so good job!

Reply:
This is true. To make sure the images load properly you must copy the image into the folder.